utils/notebook_utils/label_matrix_helper.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import numpy as np
+from scipy.stats import norm
+import scipy.sparse as sparse
+
+def get_columns(session, L_data, lf_hash, lf_name):
+    """
+    This function is designed to extract the column positions of
+    each individual label function given their category (i.e. CbG_DB or DaG_TEXT ...)
+    
+    returns a list of column positions that corresponds to each label function 
+    """
+    return list(
+    map(lambda x: L_data.key_index[x[0]],
+    session.query(L_data.annotation_key_cls.id)
+         .filter(L_data.annotation_key_cls.name.in_(list(lf_hash[lf_name].keys())))
+         .all()
+        )
+    )
+
+
+def get_auc_significant_stats(data_df, model_aucs):
+    """
+    This function is designed test the hypothesis that 
+    given aurocs are greater than 0.5 (random)
+
+    relevant websites:
+     https://www.quora.com/How-is-statistical-significance-determined-for-ROC-curves-and-AUC-values
+     https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test
+
+    returns a dataframe with the provided statsitics
+    """
+
+    class_df = data_df.curated_dsh.value_counts()
+    n1 = class_df[0]
+    n2 = class_df[1]
+    mu = (n1*n2)/2
+    sigma_u = np.sqrt((n1 * n2 * (n1+n2+1))/12)
+    print("mu: {:f}, sigma: {:f}".format(mu, sigma_u))
+
+    model_auc_df = (
+        pd.DataFrame
+        .from_dict(model_aucs, orient='index')
+        .rename(index=str, columns={0:'auroc'})
+    )
+
+    model_auc_df['u'] = model_auc_df.auroc.apply(lambda x: x*n1*n2)
+    model_auc_df['z_u'] = model_auc_df.u.apply(lambda z_u: (z_u- mu)/sigma_u)
+    model_auc_df['p_value'] = model_auc_df.z_u.apply(lambda z_u: norm.sf(z_u, loc=0, scale=1))
+
+    return model_auc_df
+
+
+def get_overlap_matrix(L, normalize=False):
```

Comment:
Better documentation of what L is... i.e. L is a matrix of labeling function scores created by snorkel.module.function.